Solaris cp doesn't know -a, but GNU does.

Not for upstream.

--- suricata-7.0.6/rust/Makefile.am
+++ suricata-7.0.6/rust/Makefile.am
@@ -51,11 +51,11 @@ $(CARGO) build $(RELEASE) $(NIGHTLY_ARGS
 			--features "$(RUST_FEATURES)" $(RUST_TARGET)
 endif
 	if test -e $(RUST_SURICATA_LIBDIR)/suricata.lib; then \
-		cp -a $(RUST_SURICATA_LIBDIR)/suricata.lib \
+		gcp -a $(RUST_SURICATA_LIBDIR)/suricata.lib \
 			$(RUST_SURICATA_LIBDIR)/${RUST_SURICATA_LIBNAME}; \
 	fi
 	if test -e $(RUST_SURICATA_LIBDIR)/libsuricata.a; then \
-		cp -a $(RUST_SURICATA_LIBDIR)/libsuricata.a \
+		gcp -a $(RUST_SURICATA_LIBDIR)/libsuricata.a \
 			$(RUST_SURICATA_LIBDIR)/${RUST_SURICATA_LIBNAME}; \
 	fi
 	$(MAKE) gen/rust-bindings.h
